contrib/adapters/ale.py

```diff
@@ -52,7 +52,7 @@ def _parse_data_line(line, columns, fps):
         if "Start" in metadata:
             try:
                 start = otio.opentime.from_timecode(metadata["Start"], fps)
-            except:
+            except Exception:
```

Comment:
Do you know which exception this would raise?  I'm assuming its a `TypeError` from `from_timecode` (since the `if "Start" in metadata` avoids the `KeyError` from the dictionary).  I would prefer that to catching `Exception`.

Reply:
Also, can do `metadata.pop("Start")` here and skip the `del` below.